### ISSUE TITLE
Error: Unexpected response from AWS  Code:  InvalidRequest

### ISSUE THREAD

**Issue by abada** [balderdashy/skipper-s3#48]
The authorization mechanism you have provided is not supported. Please use AWS4-HMAC-SHA256

**Comment by sailsbot** [comment]
@abada Thanks for posting, we'll take a look as soon as possible.  In the meantime, if you haven’t already, please carefully read the [issue contribution guidelines](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md#ii-issue-contributions) and double-check for any missing information above.  In particular, please ensure that this issue is about a stability or performance bug with a  documented feature; and make sure you’ve included detailed instructions on how to reproduce the bug from a clean install.   Finally, don’t forget to include the version of Node.js you tested with, as well as your version of Sails or Waterline, and of any relevant standalone adapters/generators/hooks.

Thank you!

**Comment by sailsbot** [comment]
@abada,@sailsbot: Hello, I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message and simply close the issue if possible.  On the other hand, if you are still waiting on a patch, please post a comment to keep the thread alive (with any new information you can provide).



If no further activity occurs on this thread within the next 3 days, the issue will automatically be closed.

Thanks so much for your help!

**Comment by glenswift** [comment]
sails team shows definitely the worst support I have ever met in open source. It's like my fifth issue without any response or attempt to solve closed but sailsbot.

**Comment by sgress454** [comment]
@glenswift Sorry about that.  The idea of the bot is to help prioritize -- in essence, to see which wheels squeak so that they (hopefully) get some grease.  In the future, if you see the "it has been 30 days" message pop up, and the issue is still valid, go ahead and post a message to that effect and it'll reset the clock.  Ideally someone else from the community will get involed -- the core team is spread a bit thin to be the only point of contact on every repo in the Sails ecosystem.  And if the bot closes an issue (especially a bug) that still needs to be addressed, do what you did here and we will re-open.

In the case of using S3 with Skipper, you might take a look at @Alaneor's [skipper-better-s3](https://www.npmjs.com/package/skipper-better-s3), which is more up-to-date.

**Comment by mikermcneil** [comment]
I'm going to go ahead and close this issue since there isn't enough information here to solve or reproduce this problem (see https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md#ii-issue-contributions).  Please feel free to open a new issue with more info.  Thanks!
